packages/ui/react-ui/src/CrossMintButtonStatus.tsx

```diff
@@ -61,21 +62,27 @@ export const CrossMintStatusButton: FC<StatusButtonProps> = ({
         }
     }, [status]);
 
+    const isClientSide = getIsClientSide();
```

Comment:
I am not 100% confident, but I think you can simplify further and 

1. Rename the exported property from getIsClientSide to isClientSide
2. export the boolean directly and not a function, as it will always return the same value given an operating environment
3. Remove this intermediate variable

Reply:
yep, it works! I hesitated a bit at first, but it definitely makes sense that it does: It will instanciate the utils file in the server, and then in the client again.